src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -15,9 +15,9 @@ import { speech_say } from '../speech';
 import { tau } from '../math/circle';
 import { abs } from '../math/core';
 import { distance2d } from '../math/distance';
-import { calcTurnInitiationDistance } from '../math/flightMath';
-import { vlen, vradial, vsub } from '../math/vector';
-import { km, kn_ms, radiansToDegrees, degreesToRadians } from '../utilities/unitConverters';
+import { isEmptyOrNotArray } from '../utilities/validatorUtilities';
+import { vlen } from '../math/vector';
+import { km } from '../utilities/unitConverters';
 import { FLIGHT_CATEGORY } from '../constants/aircraftConstants';
 import { isEmptyOrNotArray } from '../utilities/validatorUtilities';
```

Comment:
looks like there is a duplicate import in here, that is why your tests were failing on the CI.  You might want to check the other files too just to be sure, or take a look at the CI and see where its failing (you can click the link at the bottom of the page).